### PR TITLE
feat(explore): allow to provide explore tools through config

### PR DESCRIPTION
**Pull request by pjungermann**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/explore-backend/package.json [hunk 1]
```diff
@@ -42,6 +43,7 @@
     "supertest": "^6.2.4"
   },
```
Comment (Rugvip) [inline plugins/explore-backend/package.json:44]: you'll need to add the `configSchema` key here as well

**Review by Rugvip — CHANGES_REQUESTED** [review]
Yep, sweet 👍

Just a minor fix, then let's :shipit: 

Overall we're leaning over towards more things in config btw. Fits nicer with the new backend system.

File: plugins/explore-backend/package.json (hunk 1 above)
Comment (pjungermann) [reply plugins/explore-backend/package.json:44]: good catch, thanks! Fixed.

**Review by Rugvip — APPROVED** [review]
:shipit:!

**Pull request merged**